src/mlpack/methods/cf/decomposition_policies/SVD_complete_method.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file SVD_complete_incremental.hpp
+ * @author Haritha Nair
+ *
+ * Implementation of the SVD complete incremental method for use in the 
+ * Collaborative Filtering.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_SVD_COMPLETE_METHOD_HPP
+#define MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_SVD_COMPLETE_METHOD_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/amf/amf.hpp>
+#include <mlpack/methods/amf/update_rules/nmf_als.hpp>
+#include <mlpack/methods/amf/termination_policies/max_iteration_termination.hpp>
+#include <mlpack/methods/amf/termination_policies/simple_residue_termination.hpp>
+
+namespace mlpack {
+namespace cf {
+
+/**
+ * Implementation of the SVD complete incremental policy
```

Comment:
Do you think you could add a short description about what this policy does, this should be useful for people who like to use the CF code.